src/models/dataStore.ts

```diff
@@ -28,13 +29,16 @@ async function saveDataStore(d2: D2, dataStoreKey: string, newValue: any): Promi
     await dataStore.set(dataStoreKey, newValue);
 }
 
+export async function getDataStoreData(d2: D2, dataStoreKey: string) {
```

Comment:
I guess this will be part of the dataStore abstraction you mentioned @matiasgarcia91 ?

Reply:
yes, I needed this first step so I wrote it here